cpp/src/arrow/flight/flight-test.cc

```diff
@@ -620,5 +659,25 @@ TEST_F(TestAuthHandler, CheckPeerIdentity) {
   ASSERT_EQ(result->body->ToString(), "user");
 }
 
+TEST_F(TestTls, DoAction) {
+  if (!client_) {
```

Comment:
Can this actually occur?

Reply:
No, this is left over from when I made the TLS tests optional, I've removed it.